src/NetAppFiles/NetAppFiles.Test/Properties/AssemblyInfo.cs

```diff
@@ -45,6 +45,6 @@
 //
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
-[assembly: AssemblyVersion("0.9.0")]
-[assembly: AssemblyFileVersion("0.9.0")]
+[assembly: AssemblyVersion("1.1.0")]
```

Comment:
You don't need to change assembly versions for test projects

Reply:
ok, I will revert this